index.html

```diff
@@ -3640,6 +3641,12 @@ <h3><code>forms</code></h3>
 }
 </pre>
 
+<p>
+    <code>uriVariables</code> are mainly for properties and events. When retrofitting an existing system, it may be necessary to use
+    <code>uriVariables</code> for actions. In general, it is recommended to avoid <code>uriVariables</code> as much as possbile when a 
+    new WoT-based system is designed.
```

Comment:
Thanks for making this an explicit statement of the specs.

Reply:
year, that would be a kind of bigger change and should be more addressed in the TD 2.0 version. I would suggest that you create a new issue and I label it with defer 2.0